src/components/PaymentMethod/PaymentMethod.js

```diff
@@ -64,8 +64,8 @@ const PaymentMethod = ({
               title={t('Add a payment method!')}
               subtitle={t('Set up a new payment method for your account')}
               withBorder
-              onClick={() =>
-                showInnerPopup({ type: POPUP_TYPES.paymentDetails })
+              onClick={
+                () => showInnerPopup({ type: POPUP_TYPES.paymentDetails }) // TODO: cover in 3DS
```

Comment:
this also should be covered :) 
It's a case when user doesn't have any payment method